tests/publiccloud/storage_perf.pm

```diff
@@ -161,11 +168,11 @@ sub run {
         # Store values in influx-db
         if ($url) {
             my $data = {
-                table => 'storage',
+                table => 'storage_1',
```

Comment:
In this first commit I named the table `storage_1`, to save verification data only and easier find those data, keeping separate the production table.
When PR will be ready to merge, the table will be named `storage_fio`.